force-app/main/default/classes/FeatureParameters.cls

```diff
@@ -60,4 +80,65 @@ public inherited sharing class FeatureParameters {
             return finder.findCount();
         }
     }
+
+    @TestVisible
+    private inherited sharing class ActiveProgramsWithEngagementsLast30 implements FeatureManagement.FeatureParameter {
+        @TestVisible
+        private QueryBuilder queryBuilder {
+            get {
+                if (queryBuilder == null) {
+                    queryBuilder = new QueryBuilder()
+                        .withSObjectType(Program__c.SObjectType)
+                        .withConditions(buildConditions());
+                }
+
+                return queryBuilder;
+            }
+            set;
+        }
+        @TestVisible
+        private Finder finder {
+            get {
+                if (finder == null) {
+                    finder = new Finder(queryBuilder);
+                }
+
+                return finder;
+            }
+            set;
+        }
+
+        public void send() {
+            final Object value = getValue();
+
+            if (value instanceof Integer) {
+                FeatureManagement.getInstance()
+                    .setPackageIntegerValue(getName(), (Integer) value);
+            }
+        }
+
+        private String getName() {
+            return DeveloperName.ACTIVE_PROGRAMS_WITH_ENGAGEMENTS_LAST30.name()
+                .remove('_');
+        }
+
+        private Object getValue() {
+            return finder.findCount();
+        }
+
+        private List<String> buildConditions() {
```

Comment:
@jjbennett, Just a thought here. We could maybe make the buildConditions method reusable by passing parameters. For this go it should be good but maybe something to think about in the future. Thoughts???

Reply:
Yea, maybe even have the add condition in the queryBuilder just add to the list instead of only allowing 1. I think that is better and can make that change quick.

Reply:
Actually, let's leave it like it is for now but maybe make the QueryBuilder better in a future iteration. Thanks for the idea!